Tasks/JavaToolInstaller/javatoolinstaller.ts

```diff
@@ -43,12 +43,12 @@ async function getJava(versionSpec: string) {
     if (version) { //This version of Java JDK is already in the cache. Use it instead of downloading again.
         console.log(taskLib.loc('Info_ResolvedToolFromCache', version));
     } else if (fromAzure) { //Download JDK from an Azure blob storage location and extract.
-        console.log(taskLib.loc('RetrievingJdkFromAzure', version));
+        console.log(taskLib.loc('RetrievingJdkFromAzure'));
         compressedFileExtension = getFileEnding(taskLib.getInput('azureCommonVirtualFile', true));
```

Comment:
Probably a good idea to get this input `taskLib.getInput('azureCommonVirtualFile', true)` only once, save it in a variable, and use it again.

Reply:
Good point. I missed that I was using it twice when I rearranged the code. I'll fix this.